### PR TITLE
https://issues.redhat.com/browse/ACM-5559 : add .yaml callout 

### PR DESCRIPTION
**Pull request by faaden123**

### CODE REVIEW

File: add-ons/cluster_wide_proxy.adoc [hunk 1]
```diff
@@ -24,16 +24,14 @@ oc -n <my-cluster-name> edit klusterletaddonconfig
...
 ----
 +
-Replace `proxy_not_secure` with the address of the proxy server for `http` requests. For example, use `http://192.168.123.145:3128`.
-+
-Replace `proxy_secure` with the address of the proxy server for `https` requests. For example, use `https://192.168.123.145:3128`. 
-+
-Replace `no_proxy` with a comma delimited list of IP addresses, hostnames, and domain names where traffic will not be routed through the proxy. For example, use `.cluster.local,.svc,10.128.0.0/14,example.com`.
+<1> Replace `proxy_not_secure` with the address of the proxy server for `http` requests. For example, use `http://192.168.123.145:3128`.
+<2> Replace `proxy_secure` with the address of the proxy server for `https` requests. For example, use `https://192.168.123.145:3128`. 
+<3> Replace `no_proxy` with a comma delimited list of IP addresses, hostnames, and domain names where traffic will not be routed through the proxy. For example, use `.cluster.local,.svc,10.128.0.0/14,example.com`.
```
Comment (dockerymick) [inline add-ons/cluster_wide_proxy.adoc]: Suggest to remove "will" to maintain consistent, present tone. 

```suggestion
<3> Replace `no_proxy` with a comma delimited list of IP addresses, hostnames, and domain names where traffic is not routed through the proxy. For example, use `.cluster.local,.svc,10.128.0.0/14,example.com`.
```

**Review by dockerymick — CHANGES_REQUESTED** [review]
Left a suggestion

**Review by oafischer — APPROVED** [review]
lgtm

**Review by dockerymick — APPROVED** [review]

**Comment by dockerymick** [comment]
Looks great! Thanks Faisa!

**Pull request merged**